src/controllers/crossCutting.controller.js

```diff
@@ -6,18 +6,18 @@ const exportFinalVideo = require("../services/exportFinalVideo");
 const getFinalVideoUrl = require("../services/getFinalVideoUrl");
 const progressStatus = require("../routes/progressStatus");
 const { TEMP_DIR_FRAMES, TEMP_DIR } = require("../constants/paths");
+const assignEditPoints = require("../services/assignEditPoints");
```

Comment:
아래와 같이 순서 변경해주시면 좋을 것 같아요!

```suggestion
const getFinalVideoUrl = require("../services/getFinalVideoUrl");
const assignEditPoints = require("../services/assignEditPoints");
const progressStatus = require("../routes/progressStatus");
const { TEMP_DIR_FRAMES, TEMP_DIR } = require("../constants/paths");
```